lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
@rwz that's probably true.

cc: @rondale-sc 

In the mean time, what part of this diff is a point of contention to be being merged?